README.md

```diff
@@ -67,11 +67,11 @@ console.log(`Found ${response.issues.nodes.length} issues!`);
 
 ## `octokit.paginateGraphql()`
 
-The `paginateGraphql` plugin adds a new `octokit.paginateGraphql()` method which accepts a function that
+The `paginateGraphql` plugin adds a new `octokit.paginateGraphql()` method which accepts a query-builder function that
```

Comment:
how about we add `octokit.graphql.paginate` instead? `octokit.graphql` is guaranteed to exist as it's defined in `@octokit/core`. In future, we could deprecate `octokit.paginate` and move it to `octokit.rest.paginate` for coherence.